src/components/mint-button/mint-button.scss

```diff
@@ -1,3 +1,5 @@
+@import '~sass/main.scss';
```

Comment:
This is a special syntax for `webpack` loader, which allows using `node_modules` or `alias` as a  root element in import path